db/seeds.rb

```diff
@@ -8,11 +8,11 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 def seed_development
```

Comment:
Seed is going to be ran every time the container is launched, right? If yes, we can't use `create` here because it will create unnecessary registries.

Reply:
we can use `first_or_initialize` instead, which is cleaner, but even `create` wont create more registries as there is the uniq constraint

Reply:
I didn't see the validation constraint. :sweat_smile:

Reply:
updated